examples/_debug/legends/color-categories-style.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```

Comment:
Please rename the example (and all of the new ones inside 'legends' to the camelCase way, so 'colorCategories' in this case.

In fact, the style helpers are the only filenames we left (by mistake or lack of time) with the old 'snake-case-naming'. We'll plan a small refactor on that, once we have the legends.

Reply:
What a bad luck I am because I have copied all the examples from the styles folder XD